tools/configure_server_settings.rb

```diff
@@ -31,8 +58,17 @@
 key = keys.pop.to_sym
 keys.each { |p| path = path[p.to_sym] }
 
+# allow user to escape if the new value's class is not the same as the original,
+# such as setting a String where it was previously an Integer
+if opts[:force]
+  puts "Change [#{opts[:path]}], old class: [#{path[key].class}], new class: [#{newval.class}]"
+elsif path[key] && path[key].class != newval.class
+  STDERR.puts "The new value's class #{newval.class} does not match the prior one's #{path[key].class}.  Use -f to force update, this may break things!"
```

Comment:
> Use -f to force update, this may break things!"

"Use `-t` to specify the type for the provided value.  Use `-f` to force changing this value.  Note, `-f` may break things!  See `-h` for examples."